tt_llk_blackhole/llk_lib/llk_math_eltwise_ternary_sfpu_params.h

```diff
@@ -0,0 +1,65 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include "ckernel_sfpu_where.h"
+#include "llk_math_eltwise_ternary_sfpu.h"
+#include "llk_sfpu_types.h"
+
+template <bool APPROXIMATE, class F, class... ARGS>
```

Comment:
```suggestion
template <bool APPROXIMATE, typename Callable, typename... Args>
```
Again, trying to follow the convention.